warehouse/schema.go

```diff
@@ -38,10 +38,10 @@ type bundleEvent struct {
 	UserEmail              string
 	UserDisplayName        string
 	UserId                 int64
-	CustomVars             string
 	LoadDomContentTime     int64
 	LoadFirstPaintTime     int64
 	LoadEventTime          int64
+	CustomVars             string
```

Comment:
I had to do this because when we transform the json to csv we hold back all the 'unknown' columns and club them all together and insert as CustomVars in the last column. Added a note so others are not confused as well! We are actively using the schema to map the csv columns to the fsexport table columns.